javascriptlib/src/mill/javascriptlib/ReactScriptsModule.scala

```diff
@@ -0,0 +1,174 @@
+package mill.javascriptlib
+
+import mill._
+import os.*
+
+// create-react-app scripts
+trait ReactScriptsModule extends TypeScriptModule {
+  def npmDevDeps: T[Seq[String]] = Task { Seq.empty[String] }
+
+  override def npmInstall: T[PathRef] = Task.Anon {
+    os.call((
+      "npm",
+      "install",
+      "--save-dev",
+      "react@18.3.1",
+      "react-dom@18.3.1",
+      "react-scripts@5.0.1",
+      "typescript@^4.9.5",
+      "web-vitals@2.1.4",
+      "@testing-library/jest-dom@^5.17.0",
+      "@testing-library/react@^13.4.0",
+      "@testing-library/user-event@^13.5.0",
+      "@types/jest@^27.5.2",
+      "@types/node@^16.18.119",
+      "@types/react@^18.3.12",
+      "@types/react-dom@^18.3.1",
+      "serve@12.0.1",
```

Comment:
Should these be in `npmDeps` or `npmDevDeps` instead?

Reply:
hmm both. I have updated it to use npmDeps and npmDevDeps.